lib/miner.go

```diff
@@ -190,6 +198,10 @@ func (desoMiner *DeSoMiner) MineAndProcessSingleBlock(threadIndex uint32, mempoo
 		return nil, fmt.Errorf("DeSoMiner._startThread: _mineSingleBlock returned nil; should only happen if we're stopping")
 	}
 
+	if blockToMine.Header.Height >= uint64(desoMiner.params.ForkHeights.ProofOfStake2ConsensusCutoverBlockHeight) {
+		return nil, fmt.Errorf("DeSoMiner._startThread: _mineSingleBlock returned a block that is past the Proof of Stake Cutover")
```

Comment:
Should we replace this with `IsPoSBlockHeight` for consistency?

Reply:
Resolved in PR https://github.com/deso-protocol/core/pull/1009